projects/sandbox/train/train/train.py

```diff
@@ -322,6 +326,25 @@ def main(
         batches_per_chunk=int(batches_per_epoch / 8),
         chunks_per_epoch=8,
         device=device,
-        preprocessor=augmentor,
+    )
+
+    # find glitch paths and initiate glitch loader
+    glitch_paths = {}
+    for ifo in ifos:
+        ifo_dir = glitch_dir / ifo
+        glitch_paths[ifo] = list(ifo_dir.iterdir())
+
+    glitch_loader = ChunkedGlitchDataset(
+        reads_per_chunk=10,
+        batches_per_chunk=int(batches_per_epoch / 4),
```

Comment:
Does the denominator need to match `chunks_per_epoch` below?

Reply:
Yeah I think `batches_per_chunk * chunks_per_epoch` (i.e. batches_per_epoch) should be the same for both the `background_loader` and `glitch_loader`